src/Airship/Internal/Decision.hs

```diff
@@ -696,7 +693,16 @@ o16 r = do
     req <- lift request
     if requestMethod req == HTTP.methodPut
         then o14 r
-        else o18 r
+        else o17 r
+
+o17 r@Resource{..} = do
+    trace "o17"
```

Comment:
Hmm… though for-GET is gorgeous, it departs pretty significantly from the Airship graph, AFACIT